integration/integration_test.go

```diff
@@ -6547,6 +6548,42 @@ func testListResourcesAcrossClusters(t *testing.T, suite *integrationTestSuite)
 	}
 }
 
+func testJoinOverReverseTunnelOnly(t *testing.T, suite *integrationTestSuite) {
+	lib.SetInsecureDevMode(true)
+	defer lib.SetInsecureDevMode(false)
+
+	// Create a Teleport instance with Auth/Proxy.
+	mainConfig := func() *service.Config {
+		tconf := suite.defaultServiceConfig()
+		tconf.Auth.Enabled = true
+
+		tconf.Proxy.Enabled = true
+		tconf.Proxy.DisableWebService = false
+		tconf.Proxy.DisableWebInterface = true
+
+		tconf.SSH.Enabled = false
+
+		return tconf
+	}
+	main := suite.NewTeleportWithConfig(t, nil, nil, mainConfig())
+	t.Cleanup(func() { require.NoError(t, main.StopAll()) })
+
+	// Create a Teleport instance with a Node.
+	nodeConfig := func() *service.Config {
+		tconf := suite.defaultServiceConfig()
+		tconf.Hostname = Host
+		tconf.Token = "token"
+
+		tconf.Auth.Enabled = false
+		tconf.Proxy.Enabled = false
+		tconf.SSH.Enabled = true
+
+		return tconf
+	}
+	_, err := main.StartNodeWithTargetPort(nodeConfig(), helpers.PortStr(t, main.ReverseTunnel))
+	require.NoError(t, err)
```

Comment:
nit: add error details to the focus of the test. It helps draw attention to what is setup vs what is under test, plus it makes an eventual error nicer.

```suggestion
	require.NoError(t, err, "Node failed to join over reverse tunnel")
```